net.jimblackler.jsonschemafriend:core dependency is not available in maven central

The dependency

```
<dependency>
  <groupId>net.jimblackler.jsonschemafriend</groupId>
  <artifactId>core</artifactId>
  <version>0.12.4</version>
</dependency>
```
is not available in maven central, which could be in conflict with individual policies on the consumer side and could render the library and downstream projects not usable in some cases.

Is there a possibility to eliminate this dependency replacing it by something that is available in maven central. Alternatively a version could be published according to well known policies on maven central. 

The published artifacts on https://maven.scijava.org show strange variations in their timestamps. We would assume that it is unclear who built and published the artifacts. Compare 
https://maven.scijava.org/content/repositories/public/net/jimblackler/jsonschemafriend/core/0.12.4/

See also:
https://github.com/jimblackler/jsonschemafriend/issues/19

Kind regards,
Karsten

Currently same issue here.

```
[ERROR] Failed to execute goal org.spdx:spdx-maven-plugin:1.0.0-RC2:createSPDX (build-spdx) on project oras-java-sdk: Execution build-spdx of goal org.spdx:spdx-maven-plugin:1.0.0-RC2:createSPDX failed: Plugin org.spdx:spdx-maven-plugin:1.0.0-RC2 or one of its dependencies could not be resolved:
```

> Currently same issue here.
> 
> ```
> [ERROR] Failed to execute goal org.spdx:spdx-maven-plugin:1.0.0-RC2:createSPDX (build-spdx) on project oras-java-sdk: Execution build-spdx of goal org.spdx:spdx-maven-plugin:1.0.0-RC2:createSPDX failed: Plugin org.spdx:spdx-maven-plugin:1.0.0-RC2 or one of its dependencies could not be resolved:
> ```

@jonesbusy do you have the full error message please? Thx

Sure my bad.

```
Downloading from jitpack.io: https://jitpack.io/net/jimblackler/jsonschemafriend/core/0.12.4/core-0.12.4.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.743 s
[INFO] Finished at: 2025-04-17T15:41:41+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.spdx:spdx-maven-plugin:1.0.0-RC2:createSPDX (build-spdx) on project oras-java-sdk: Execution build-spdx of goal org.spdx:spdx-maven-plugin:1.0.0-RC2:createSPDX failed: Plugin org.spdx:spdx-maven-plugin:1.0.0-RC2 or one of its dependencies could not be resolved:
[ERROR]         Could not find artifact net.jimblackler.jsonschemafriend:core:jar:0.12.4 in jitpack.io (https://jitpack.io)
```

--- a/pom.xml
+++ b/pom.xml
@@ -57,12 +57,6 @@
         <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
       </repository>
 	</distributionManagement>
-	<repositories>
-      <repository>
-        <id>jitpack.io</id>
-        <url>https://jitpack.io</url>
-      </repository>
-    </repositories>
 	<profiles>
 		<profile>
 			<id>release</id>
@@ -151,9 +145,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>net.jimblackler.jsonschemafriend</groupId>
-      <artifactId>core</artifactId>
-      <version>0.12.3</version>
+      <groupId>org.metaeffekt.bundle.jsonschemafriend</groupId>
+      <artifactId>ae-jsonschemafriend-core</artifactId>
+      <version>0.12.5-1</version>
     </dependency>
     <dependency>
     	<groupId>org.spdx</groupId>

